packages/sanity/src/core/form/inputs/DateInputs/base/calendar/CalendarMonth.tsx

```diff
@@ -11,13 +10,22 @@ interface CalendarMonthProps {
   selected?: Date
   onSelect: (date: Date) => void
   hidden?: boolean
+  weekDayNames: [
```

Comment:
Given these are short names, should the prop name indicate that? In case we need full names at some point?

Reply:
Thanks! Fixed